packages/react-charting/docs/TestPlans/LineChart/ComponentTests.md

```diff
@@ -0,0 +1,67 @@
+**Line Chart – Component test plan**
+
+**Sub-components: Line, x-axis, y-axis, Event, Time Range, Callout, Drop down and Legend**
+
+1. **Line: Line data, Line color (multi colors), Line label (show/hide)**
+1. **Legends: show/hide legends, highlight the corresponding line on legend hover**
+1. **Callout: Default/custom callout**
+1. **Labels: x-Axis labels**
+
+|                                      **Test steps**                                       |                                                             **Validation**                                                              | **Tool used** |
+| :---------------------------------------------------------------------------------------: | :-------------------------------------------------------------------------------------------------------------------------------------: | :-----------: |
+|                                Test 1: [Snapshot testing]                                 |                                                                                                                                         |               |
+|                     - With only data prop, numerical data on x-axis.                      |                                                      Renders line chart correctly                                                       |      RTL      |
```

Comment:
Add a test case with string data on x axis. The chart should not work in this case.